src/types/HealthService.ts

```diff
@@ -0,0 +1,5 @@
+import type { HealthStatus } from '../values';
+
+export interface HealthService {
+  getHealth: () => HealthStatus;
```

Comment:
Hm, can you say more about the motivation for having this `interface` defined in a separate file? I feel like every change to this file is going to necessarily be accompanied by a change to the implementation, and having them in separate files will be more work than it needs to be.

Also, it turns out the only use of this interface is in the definition of its implementation; unless and until we actually have a need for it, maybe it'd be simpler to omit it entirely and rely on TypeScript's inference.

Reply:
You nailed it -- Java instincts plus the training to only have one class per file via eslint -- happy to follow the pattern you describe.

Side note around inference vs explicit: I'd love to have a quick 1:1 with you tomorrow to show you what I was getting from the linter before adding an explicit class for the controller specifically, in case I was just doing something silly and went overboard on the fix.

Reply:
Update: the only reason I defined these interfaces is becuse the linter was upset at the line:

`status: healthService.getHealth(),`

saying that `getHealth` was being called on an `any` typed variable.

This no longer happens, even after removing the explicit interface definitions, so I'm just going to move along.

Pushing the fix now!